Mathlib/Analysis/Normed/Group/Basic.lean

```diff
@@ -5,7 +5,8 @@ Authors: Patrick Massot, Johannes Hölzl, Yaël Dillies
 -/
 import Mathlib.Analysis.Normed.Group.Seminorm
 import Mathlib.Data.NNReal.Basic
-import Mathlib.Topology.MetricSpace.Basic
+import Mathlib.Topology.Algebra.Ring.Real
```

Comment:
This is a bit of an import increase: the question is whether this is desirable or not.